internal/pkg/cli/env_init.go

```diff
@@ -442,22 +447,22 @@ func (o *initEnvOpts) askAdjustResources() error {
 	return nil
 }
 
-func (o *initEnvOpts) importVPCConfig() *deploy.ImportVPCConfig {
+func (o *initEnvOpts) importVPCConfig() *template.ImportVPCOpts {
```

Comment:
If the goal is to get rid of the methods in the `deploy` pkg which was nice, instead we can have the template pkg depend on the config pkg. I think it makes more sense for other packages depend on config rather than the other way around since that's the source of truth for us

Reply:
it is difficult to reverse the dependency because the `config` pkg is depending on the `template` pkg in various places. Would you mind me to do it in a following PR?

Reply:
really?? That's so weird ok sure!

Reply:
you are correct lol. that was `deploy` pkg.